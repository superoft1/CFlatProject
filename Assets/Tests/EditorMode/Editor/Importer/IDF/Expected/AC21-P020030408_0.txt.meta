fileFormatVersion: 2
guid: 414b4c212b25aa949ba6e33ff45455c6
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
