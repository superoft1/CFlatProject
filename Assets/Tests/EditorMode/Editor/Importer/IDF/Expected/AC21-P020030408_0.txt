CheckValve, P83C-002X8, (0.0, -1.0, 0.0), (-11943.9, 9266.5, 100.9), True
(-11943.9, 9266.2, 100.9)
(-11943.9, 9266.8, 100.9)
FreeVertex: 1
WeldNeckFlange, P61B-003LX, (0.0, 1.0, 0.0), (-11943.9, 9266.1, 100.9), False
(-11943.9, 9266.2, 100.9)
(-11943.9, 9266.1, 100.9)
FreeVertex: 0
Pipe, P01A-0016U, (0.0, -1.0, 0.0), (-11943.9, 9266.0, 100.9), False
(-11943.9, 9266.1, 100.9)
(-11943.9, 9265.9, 100.9)
FreeVertex: 0
WeldOlet, P01A-0016U, (0.0, 0.0, -1.0), (-11943.9, 9265.9, 100.9), False
(-11943.9, 9265.9, 100.9)
(-11943.9, 9265.9, 100.9)
(-11943.9, 9265.9, 100.8)
FreeVertex: 0
GateValve, P81B-00393, (0.0, 0.0, -1.0), (-11943.9, 9265.9, 100.6), False
(-11943.9, 9265.9, 100.4)
(-11943.9, 9265.9, 100.8)
FreeVertex: 0
Cap, P29B-0000D, (0.0, 0.0, -1.0), (-11943.9, 9265.9, 100.4), False
(-11943.9, 9265.9, 100.4)
FreeVertex: 0
Pipe, P01A-0016U, (0.0, -1.0, 0.0), (-11943.9, 9265.9, 100.9), False
(-11943.9, 9265.9, 100.9)
(-11943.9, 9265.9, 100.9)
FreeVertex: 0
WeldOlet, P01A-0016U, (0.0, 0.0, 1.0), (-11943.9, 9265.9, 100.9), False
(-11943.9, 9265.9, 100.9)
(-11943.9, 9265.9, 100.9)
(-11943.9, 9265.9, 101.1)
FreeVertex: 1
Pipe, P01A-0016U, (0.0, -1.0, 0.0), (-11943.9, 9265.8, 100.9), False
(-11943.9, 9265.9, 100.9)
(-11943.9, 9265.7, 100.9)
FreeVertex: 0
WeldNeckFlange, P61B-003LX, (0.0, -1.0, 0.0), (-11943.9, 9265.6, 100.9), False
(-11943.9, 9265.5, 100.9)
(-11943.9, 9265.7, 100.9)
FreeVertex: 0
OpenSpectacleBlank, P71B-000B5, (0.0, -1.0, 0.0), (-11943.9, 9265.5, 100.9), False
(-11943.9, 9265.5, 100.9)
(-11943.9, 9265.5, 100.9)
FreeVertex: 0
GateValve, P81C-005ZS, (0.0, -1.0, 0.0), (-11943.9, 9265.3, 100.9), False
(-11943.9, 9265.0, 100.9)
(-11943.9, 9265.5, 100.9)
FreeVertex: 0
WeldNeckFlange, P61B-003LX, (0.0, 1.0, 0.0), (-11943.9, 9265.0, 100.9), False
(-11943.9, 9265.0, 100.9)
(-11943.9, 9264.9, 100.9)
FreeVertex: 0
Pipe, P01A-0016U, (0.0, -1.0, 0.0), (-11943.9, 9264.8, 100.9), False
(-11943.9, 9264.9, 100.9)
(-11943.9, 9264.7, 100.9)
FreeVertex: 0
WeldOlet, P01A-0016U, (0.0, 0.0, 1.0), (-11943.9, 9264.7, 100.9), False
(-11943.9, 9264.7, 100.9)
(-11943.9, 9264.7, 100.9)
(-11943.9, 9264.7, 101.1)
FreeVertex: 1
Pipe, P01A-0016U, (0.0, -1.0, 0.0), (-11943.9, 9264.1, 100.9), False
(-11943.9, 9264.7, 100.9)
(-11943.9, 9263.6, 100.9)
FreeVertex: 0
Pipe, P01A-0016U, (0.0, -1.0, 0.0), (-11943.9, 9263.5, 100.9), False
(-11943.9, 9263.6, 100.9)
(-11943.9, 9263.4, 100.9)
FreeVertex: 0
WeldNeckFlange, P61B-003LX, (0.0, -1.0, 0.0), (-11943.9, 9263.3, 100.9), False
(-11943.9, 9263.3, 100.9)
(-11943.9, 9263.4, 100.9)
FreeVertex: 0
DiaphramControlValve, 020-UZV-4007, (0.0, -1.0, 0.0), (-11943.9, 9263.0, 100.9), False
(-11943.9, 9262.7, 100.9)
(-11943.9, 9263.2, 100.9)
FreeVertex: 1
FreeVertexTotal: 4
